test/e2e/e2e.go

```diff
@@ -175,6 +191,64 @@ var _ = Describe("validation", func() {
 			})
 		})
 	})
+
+	Context("Creating AddressPool", func() {
+		It("Should create addresspool CR successfully", func() {
+			By("By creating a configmap")
```

Comment:
this name is confusing, we expect to see a configmap created, we're not creating it ourselves

Reply:
correct will rename